components/table/index.en-US.md

```diff
@@ -187,6 +187,7 @@ One of the Table `columns` prop for describing the table's columns, Column has t
 | defaultSortOrder | Default order of sorted values | `ascend` \| `descend` | - |  |
 | ellipsis | The ellipsis cell content, not working with sorter and filters for now.<br />tableLayout would be `fixed` when `ellipsis` is `true` or `{ showTitle?: boolean }` | boolean \| {showTitle?: boolean } | false | showTitle: 4.3.0 |
 | filterDropdown | Customized filter overlay | ReactNode \| (props: [FilterDropdownProps](https://github.com/ant-design/ant-design/blob/ecc54dda839619e921c0ace530408871f0281c2a/components/table/interface.tsx#L79)) => ReactNode | - |  |
+| filterDropdownProps | Customized dropdown props | [DropdownProps](https://ant.design/components/dropdown#api) | - |  |
```

Comment:
How about add original dropdown node to filterDropdown?

```jsx
filterDropdown: (props, originalNode) => {
  return originalNode;
}
```

```jsx
filterDropdown: (props, originalNode) => {
  return React.cloneElement(originalNode, { ...dropdownProps });
}
```

Reply:
It is better since we won't have to add a prop, but I'm not sure about it's type checking. I'll have a look

Reply:
Wow great, i thought this was forgotten. I am still willing to implement it but you guys should decide on the solution.

Personally i like adding **filterDropdownProps** and removing **filterDropdownOpen** prop. but if you guys prefer adding **originalNode** i'm okay with that too, just tell me where to put it for the next major version